images/kube-proxy/OWNERS

```diff
@@ -0,0 +1,5 @@
+reviewers:
```

Comment:
@pecameron could you add people from the net team to the image OWNERS too? As long as @smarterclayton thinks that's OK of course. To both Reviewers and Approvers.

Reply:
Done @dcbw @squeed PTAL